distributions/openhab/src/main/resources/bin/backup

```diff
@@ -55,9 +55,15 @@ setup(){
 
   fileList="$OPENHAB_RUNTIME/bin/userdata_sysfiles.lst"
 
+  ## Parse arguments
+  if [ "$1" = "--full" ] || [ "$2" = "--full" ]; then
+    echo "including cache"
+    INCLUDE_CACHE="true"
+  fi
+
   timestamp=$(date +"%y_%m_%d-%H_%M_%S")
   ## Set the filename
-  if [ -z "$1" ]; then
+  if [ -z "$1" ] || [ "$1" = "--full" ]; then
```

Comment:
This can stay as it was if the above comment is taken onboard. Otherwise, `./runtime/bin/backup --full /my/directory` will not do as a user expects.